app/google-services.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
This file is still not removed

Reply:
that file is needed for the Firebase Analytics